cluster.go

```diff
@@ -1056,16 +1066,69 @@ func (c *Cluster) pin(pin api.Pin, blacklist []peer.ID, prioritylist []peer.ID)
 // of underlying IPFS daemon unpinning operations.
 func (c *Cluster) Unpin(h *cid.Cid) error {
 	logger.Info("IPFS cluster unpinning:", h)
+	cState, err := c.consensus.State()
```

Comment:
I am not 100% sure this (the `Cluster` component) is the right place to have this logic. Might as well be part of LogOp Apply(), although I see advantages of keeping it here. Let me think about this until next round.

Reply:
Let me know if you would prefer this in Apply().